pkg/build/nodeimage/internal/container/docker/archive.go

```diff
@@ -26,15 +26,21 @@ import (
 	"os"
 	"strings"
 
+	"github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"sigs.k8s.io/kind/pkg/errors"
 )
 
+// ioContainerdImageName appears as an annotation in a manifest entry in "index.json".
+const ioContainerdImageName = "io.containerd.image.name"
```

Comment:
why do we have containerd annotations here? we're processing docker images?

Reply:
Because Docker maintains compatibility with containerd